src/components/16-homepage/homepage.config.yml

```diff
@@ -0,0 +1,15 @@
+label: Homepage
+status: wip
+preview: "@sam-content"
+variants:
+  - name: tag
+    label: SDS Offical Tag
+    context:
+      label:
+        classes: "sds-official-tag"
+
+  - name: tile
+    label: SDS Tile
+    context:
+      label:
+        classes: "sds-tile"
```

Comment:
could we remove the homepage grouping and have individual components for tag and title, as discussed yesterday